### PR TITLE
VMI Details tab - scheduling section

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

## 📝 Description

> Added VMI Details tab - scheduling section 

TODO:
Flavor modal, Flavor memory size

## 🎥 Demo

image

### CODE REVIEW

**Comment by metalice** [comment]
@vojtechszocs @pcbailey @glekner @hstastna @avivtur @yaacov @upalatucci  PTAL

File: src/views/virtualmachinesinstance/details/tabs/details/components/Scheduling/Flavor/Flavor.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+
+import { getCPUCount, getFlavor } from '../../../utils/VirtualMachinesInstancePageDetailsTabUtils';
```
Comment (glekner) [inline src/views/virtualmachinesinstance/details/tabs/details/components/Scheduling/Flavor/Flavor.tsx:6]: please add selectors and shared utils to `src/utils/resource/vmi`

File: src/views/virtualmachinesinstance/details/tabs/details/virtual-machines-instance-details-tab.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.VirtualMachinesInstanceDetailsTab {
+  .link-icon {
```
Comment (glekner) [inline src/views/virtualmachinesinstance/details/tabs/details/virtual-machines-instance-details-tab.scss:2]: no need for custom css.

you can take classnames from console

for example
`<PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />`

File: src/views/virtualmachinesinstance/details/tabs/details/utils/VirtualMachinesInstancePageDetailsTabUtils.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { V1CPU } from '@kubevirt-ui/kubevirt-api/kubevirt';
```
Comment (glekner) [inline src/views/virtualmachinesinstance/details/tabs/details/utils/VirtualMachinesInstancePageDetailsTabUtils.ts:1]: @metalice I we can use these selectors/utils in other places like the wizard's scheduling tab.
please move those to the `src/utils`

Comment (metalice) [reply src/views/virtualmachinesinstance/details/tabs/details/utils/VirtualMachinesInstancePageDetailsTabUtils.ts]: moved to utils

**Pull request merged**